Source/Core/CircleGeometry.js

```diff
@@ -73,7 +70,10 @@ define([
             vertexFormat : options.vertexFormat,
             stRotation : options.stRotation
         };
-        this._ellipseGeometry = new EllipseGeometry(ellipseGeometryOptions);
+        this._ellipseGeometry = undefined;
+        if (radius > 0) {
+            this._ellipseGeometry = new EllipseGeometry(ellipseGeometryOptions);
```

Comment:
Gah, I just realized this won't work because it'll break `pack` and `unpack`, which we need for our web workers.
For simplicity, let's just let EllipseGeometry handle the undefined case.  Revert this line and get rid of the check in `CircleGeometry.createGeometry`